src/tests/index.tsx

```diff
@@ -10,8 +10,48 @@ import { defaultLocale } from 'context/locale/defaultLocale';
 import { LocaleContext } from 'context/locale/localeContext/LocaleContext';
 import { AuthContext } from 'context/auth/authContext/AuthContext';
 import { ApiClientContextController } from 'context/apiClient/apiClientContextController/ApiClientContextController';
+import { AppRoute } from 'routing/AppRoute.enum';
+
+type RouterConfig =
+  | {
+      withRouter: true;
+      routerHistory: string[];
+      path: AppRoute;
+    }
+  | { withRouter: false };
+
+type ExtraRenderOptions = {
+  routerConfig?: RouterConfig;
+};
+
+type WrapperProps = {
+  children: ReactNode;
+  routerConfig?: RouterConfig;
+};
```

Comment:
you could move those types to separate file 😁

Reply:
thanks, I forgot